add_codeowners.py

```diff
@@ -98,14 +98,18 @@ def from_dict(cls, source: dict):
         return settings
 
 
-GROUP_TO_EXT: Dict[LanguageOwners, str] = {
-    LanguageOwners.PYTHON: '*.py*',
-    LanguageOwners.TWINCAT: '*.{tsproj,plcproj,tmc,tpr,xti,TcTTO,TcPOU,TcDUT,TcGVL,'
-                            'TcVis,TcVMO,TcGTLO}',
-    LanguageOwners.C: '*.{c,cpp,cc,h,h++,hh,hpp}',
-    LanguageOwners.SHELL: '*.{sh,zsh,csh,bash}',
-    LanguageOwners.EPICS: '*.{archive,autosave,cmd,db,dbd,edl,ioc,proto,req,'
-                          'sub-arch,sub-req,substitutions,tpl-arch,tpl-req}',
+GROUP_TO_EXT: Dict[LanguageOwners, list[str]] = {
+    LanguageOwners.PYTHON: ['*.py', '*.pyi', '*.pyc', '*.pyw', '*.pyx', '*.pyd'],
+    LanguageOwners.TWINCAT: ['*.tsproj', '*.plcproj', '*.tmc', '*.tpr', '*.xti',
+                             '*.TcTTO', '*.TcPOU', '*.TcDUT', '*.TcGVL', '*.TcVis',
+                             '*.TcVMO', '*.TcGTLO'],
+    LanguageOwners.C: ['*.c', '*.cpp', '*.cc', '*.h', '*.h++', '*.hh', '*.hpp'],
+    LanguageOwners.SHELL: ['*.sh', '*.zsh', '*.csh', '*.bash'],
+    LanguageOwners.EPICS: ['*.archive', '*.autosave', '*.cmd', '*.db', '*.dbd',
+                           '*.edl', '*.ioc', '*.proto', '*.req',
+                           '*.sub-arch', '*.sub-req', '*.substitutions',
+                           '*.tpl-arch', '*.tpl-req', 'RELEASE_SITE',
+                           '/configure/RELEASE'],
```

Comment:
I was wondering about including stuff like https://github.com/pcdshub/ioc-common-gigECam/blob/master/configure/Makefile in here but maybe at some point these epics-centric repos that use all the random stub files default to EPICS language owners anyway. So this is more to catch when EPICS devs are needed in non-EPICS repos, in which case it is plenty thorough.

Reply:
This is a good way of framing it, might actually require some changes to our specification.  

If sme and area owners are specified, they will be listed as the `*` default group.  We don't use this logic for language owners.  Should we adjust our logic here?  Or add an EPICS SME role that's separate from the language expertise?d

Reply:
I wonder if EPICS is special or if there are lots of "core" libraries in various SME groups that make sense to break out in this way

Reply:
We are sort of using "python-sme" as shorthand for "the python stack", so maybe that could use some refinement as well.  It's hard to separate, truthfully.  Part of being a codeowner/reviewer is understanding the ecosystem around the code, so I wouldn't necessarily be comfortable assigning someone as an EPICS reviewer if they just knew syntax and not the modules/ecosystem.

But at the same time, this prevents us from accounting for random pythnon code in EPICS repos, and visa versa.  I imagine a "python-stack-owners" and "python-reviewers", might just end up being the same reviewers for us, but maybe one day the'll be different 🤷 f

Reply:
If we're not entirely sure, I'd default to trying to break out sme teams for sme roles we track internally. Though, as you say, functionally the "bluesky" smes and the "hutch python" smes and the existing "python-sme" are more or less the same people.